### DISCUSSION TITLE
Yarn 2+ seems to run commands into separate jobs/shell

### DISCUSSION THREAD

**Discussion by Sushi21** [yarnpkg/berry#5486] [Q&A] [answered]
I used to have a package.json script like this.

"test:qunit": "set \"NODE_OPTIONS=--openssl-legacy-provider\" & set \"NODE_ENV=test\" & node ./tasks/execute test/qunit",

But since I upgraded to Yarn 2+. It doesn't apply the set anymore in the same shell as the node task.

And ending having something like this in my console.

Job [2], 'set NODE_ENV=test' has ended
Job [1], 'set NODE_OPTIONS=--openssl-legacy-provider' has ended
....

Am I missing something?? Is there a way to tell it to run everything in the same shell?

**Comment by arcanis** [comment, marked as answer]
Yarn 2+ runs all scripts through a posix-like interpreter, for portability across systems. Replace your command by something like:

```
"test:qunit": "NODE_OPTIONS=--openssl-legacy-provider NODE_ENV=test node ./tasks/execute test/qunit",
```

**Reply by Sushi21** [reply]
Do I have to install a package or apply a specific syntax for Windows?

Edit: It is actually working on windows!!
Thanks a lot!!!
